.travis.yml

```diff
@@ -77,6 +77,7 @@ env:
 
 before_install:
   # Remove Ruby directive from Gemfile as this image has 2.2.5
+  - nvm install 6.10.2
```

Comment:
the comment above about Ruby, relates to the line below about Ruby.  And in the middle you've added Node ...